rides-android/build.gradle

```diff
@@ -43,24 +44,31 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_7
         targetCompatibility JavaVersion.VERSION_1_7
     }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
```

Comment:
Required updating to Gradle 3.3.0.  (Tests using R.strings.xyz started failing on upgrade.)